tests/Unit/ArticleLinkSuggestionTest.php

```
@@ -0,0 +1,108 @@
+            'anchor_text' => 'altro termine',
+            'reason' => 'altro motivo',
+            'confidence_score' => 60,
+        ]);
+    }
+
+    // 3. Cancellare l'articolo sorgente o target cancella i suggerimenti collegati
+    public function test_deleting_either_article_cascades_to_the_suggestion(): void
+    {
+        $source = $this->article();
+        $target = $this->article();
+        ArticleLinkSuggestion::create([
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\ArticleLinkSuggestion::create().


(staticMethod.notFound)